src/Symfony/Bridge/Twig/Mime/NotificationEmail.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use Symfony\Component\ErrorRenderer\Exception\FlattenException;
+use Symfony\Component\Mime\Header\Headers;
+use Symfony\Component\Mime\Part\AbstractPart;
+use Twig\Extra\CssInliner\CssInlinerExtension;
+use Twig\Extra\Inky\InkyExtension;
+use Twig\Extra\Markdown\MarkdownExtension;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class NotificationEmail extends TemplatedEmail
+{
+    public const URGENT = 'urgent';
+    public const HIGH = 'high';
+    public const MEDIUM = 'medium';
+    public const LOW = 'low';
+
+    private static $htmlTemplatePath = '@email/notification.html.twig';
+    private static $textTemplatePath = '@email/notification.txt.twig';
+
+    private $context = [
+        'importance' => 'low',
+        'content' => '',
+        'exception' => false,
+        'action_url' => null,
+        'action_text' => null,
+        'markdown' => false,
+        'raw' => false,
+    ];
+
+    public function __construct(Headers $headers = null, AbstractPart $body = null)
+    {
+        if (!class_exists(CssInlinerExtension::class)) {
+            throw new \LogicException(sprintf('You cannot use "%s" if the CSS Inliner Twig extension is not available; try running "composer require twig/cssinliner-extra".', static::class));
+        }
+
+        if (!class_exists(InkyExtension::class)) {
+            throw new \LogicException(sprintf('You cannot use "%s" if the Inky Twig extension is not available; try running "composer require twig/inky-extra".', static::class));
+        }
+
+        parent::__construct($headers, $body);
+    }
+
+    /**
+     * @return $this
+     */
+    public function markdown()
```

Comment:
IMHO this should accept a string (the markdown content)

Reply:
Done